docs/Rootless.md

```diff
@@ -80,3 +80,14 @@ else
 	TARGET = iphone:clang:latest:7.0
 endif
 ```
+
+### ``rootless.h`` implementation
```

Comment:
Taking another look at the rootless doc, it may make more sense to move these example implementations up to the relevant item (e.g., the conditional check would go under the scheme bullet and this would go under the rootless.h bullet).